docs/sources/reference/components/otelcol.connector.servicegraph.md

```diff
@@ -61,6 +61,7 @@ Name                        | Type             | Description
 `store_expiration_loop`     | `duration`       | The time to expire old entries from the store periodically.         | `"2s"`       | no
 `metrics_flush_interval`    | `duration`       | The interval at which metrics are flushed to downstream components. | `"0s"`       | no
 `database_name_attribute`   | `string`         | The attribute name used to identify the database name from span attributes. | `"db.name"`  | no
+`virtual_node_peer_attributes` | `list(string)` | List of attributes whose presence in client span will result in creation of a virtual server node | `["peer.service", "db.name", "db.system"]` | no
```

Comment:
After the table I would add that the attributes are ordered by priority and that an empty list disables virtual node creation.

Reply:
Done in HEAD :D